source/iostreams.tex

```diff
@@ -6029,7 +6114,7 @@
 defines a class that is responsible for doing exception safe prefix and suffix
 operations.
 
-\indexlibrary{\idxcode{sentry}!constructor}%
```

Comment:
Why are you deleting this here, but you left both "sentry" and "basic_ostream::sentry" for the destructor?